lib/msal-common/src/utils/Constants.ts

```diff
@@ -195,23 +191,16 @@ export type GrantType = (typeof GrantType)[keyof typeof GrantType];
 /**
  * Account types in Cache
  */
-export const CacheAccountType = {
```

Comment:
Same here

Reply:
We aren't using this for typing so i decomposed it.